### PR TITLE
Family members scan summary

### PR DESCRIPTION
**Pull request by chennara**
Screenshot_1701257413

### CODE REVIEW

File: src/scan/familyCheckinSummary/icons/PointsCollectedIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import * as Styled from './style';
+
+type TProps = {
+  count: number;
+};
+
+export const PointsCollectedIcon = ({ count }: TProps) => {
```
Comment (samvanhoeyicapps) [inline src/scan/familyCheckinSummary/icons/PointsCollectedIcon.tsx:7]: could you use the same as on ScanSucces?

File: src/scan/familyCheckinSummary/icons/style.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  width: 32px;
+  height: 32px;
+  border-radius: 16px;
+  border: ${({ type, theme }) => `4px solid ${type === 'success' ? theme.palette.primary['200'] : theme.palette.error['200']}`};
+  justify-content: center;
+  align-items: center;
+`;
+
+export const Label = styled(Typography)`
+  line-height: 24px; // $height - $borderWidth
```
Comment (samvanhoeyicapps) [inline src/scan/familyCheckinSummary/icons/style.ts:15]: you could also calculate this

File: src/scan/familyCheckinSummary/style.ts [hunk 3]
```diff
@@ -1,11 +1,34 @@
...
   flex: 1;
+  margin-top: 40px;
+`;
+
+export const FamilyMembersPoints = styled(FamilyMembersPointsComp)`
+  padding-horizontal: 16px;
 `;
 
 export const CloseButton = styled(Button)`
   margin: 16px;
```
Comment (samvanhoeyicapps) [inline src/scan/familyCheckinSummary/style.ts:32]: you could also do: margin: 16px 16px 0px 16px

File: src/scan/familyCheckinSummary/icons/PointsCollectedIcon.tsx (hunk 1 above)
Comment (chennara) [reply src/scan/familyCheckinSummary/icons/PointsCollectedIcon.tsx]: The icon in `ScanSuccess` seems to be much bigger than the one we need here.

**Review by samvanhoeyicapps — APPROVED** [review]

**Pull request merged**